modules/programs/waybar.nix

```
@@ -241,12 +197,17 @@ in {
     systemd.enable = mkEnableOption "Waybar systemd integration";
 
     style = mkOption {
-      type = nullOr str;
+      type = nullOr (oneOf [ path str ]);
```

When there are only two elements I think either is a bit prettier:
```suggestion
      type = nullOr (either path str);
```